components/doc/splitbutton/outlineddoc.js

```diff
@@ -53,6 +53,7 @@ export function OutlinedDoc(props) {
 <SplitButton label="Save" icon="pi pi-plus" onClick={save} model={items} severity="warning" outlined />
 <SplitButton label="Save" icon="pi pi-plus" onClick={save} model={items} severity="help" outlined />
 <SplitButton label="Save" icon="pi pi-plus" onClick={save} model={items} severity="danger" outlined />
+<SplitButton label="Save" icon="pi pi-plus" onClick={save} model={items} severity="contrast" outlined />
```

Comment:
I wonder if `contrast` should be an option added to this PR for all items?  https://github.com/primefaces/primereact/pull/5765